wpeverest/everest-forms
# Enhance - Form builder validation

### All Submissions:






### Changes proposed in this Pull Request:

List required validation in the form builder
1. Number Field:  step > 0
2. File Upload: max file number > 0
3.  Range: step > 0

### How to test the changes in this Pull Request:

1. Add the above fields in the form .
2.  Increase/Decrease steps value.
3. Verify if it allows only positive number.

### Types of changes:




### Other information:




### Changelog entry

>Enhance - Form builder validation.
